### PR TITLE
list appointments

### LINKED ISSUE #53: List appointments
Ability for the users to list out all the appointments in the database.

### PR DESCRIPTION
**Pull request by tohpinren**
Changes:
- Add Patient field to Appointment
- Add storing of Appointments (using a UniqueAppointmentList)
- Edit add appointment command to save to list
- Edit frontend to show appointments panel
- Rename files to fit ClinicAssistant

Todo:
- Fix storage for appointments and patients

### CODE REVIEW

**Comment by simbayippy** [comment]
PR created for fixes on your repo

File: src/main/java/seedu/address/logic/commands/EditAppointmentCommand.java [hunk 1]
```diff
@@ -90,7 +90,8 @@ public CommandResult execute(Model model) throws CommandException {
 
         // get the appointment to edit from the list of appointments of Person
         Appointment appointmentToEdit = appointmentList.get(appointmentIndex.getZeroBased());
-        Appointment editedAppointment = createEditedAppointment(appointmentToEdit, editAppointmentDescriptor);
+        Appointment editedAppointment = createEditedAppointment(appointmentToEdit, editAppointmentDescriptor,
+                editedPerson);
 
         editedPerson.editAppointment(appointmentIndex.getZeroBased(), editedAppointment);
```
Comment (simbayippy) [inline src/main/java/seedu/address/logic/commands/EditAppointmentCommand.java:96]: currently after edit_appt command, FE does not update. Same situation for delete_appt

**Review by simbayippy — CHANGES_REQUESTED** [review]
FE for Appointment list is not updating after edit_appt / delete_appt

**Comment by simbayippy** [comment]
> FE for Appointment list is not updating after edit_appt / delete_appt

also checkstyle warnings

**Comment by tohpinren** [comment]
@simbayippy This is for listing appointments only - Not edit and delete will update in future issues. Will fix checkstyle

**Comment by simbayippy** [comment]
> @simbayippy This is for listing appointments only - Not edit and delete will update in future issues. Will fix checkstyle

okie got it

**Review by simbayippy — APPROVED** [review]
LGTM

**Pull request merged**